packages/beacon-node/src/api/impl/validator/index.ts

```diff
@@ -335,6 +338,36 @@ export function getValidatorApi({
       );
   }
 
+  async function updateDefaultGraffiti(): Promise<void> {
+    defaultGraffiti = await getDefaultGraffiti();
+  }
+
+  async function getDefaultGraffiti(): Promise<string> {
+    const consensusCode = ClientCode.LS;
+    const consensusCommit = opts.version ?? "";
+
+    const lodestarClientVersion = {
+      code: consensusCode,
+      name: "Lodestar",
+      version: opts.version ?? "",
+      commit: consensusCommit,
+    };
+
+    const executionClientVersions = await chain.executionEngine
+      .getClientVersion(lodestarClientVersion)
+      .catch((_) => []);
+
+    if (executionClientVersions.length !== 0) {
+      const executionCode = executionClientVersions[0].code;
+      const executionCommit = executionClientVersions[0].commit;
+
+      return `${executionCode}|${executionCommit.slice(0, 2)}|${consensusCode}|${consensusCommit.slice(0, 2)}`;
```

Comment:
Is this format defined somewhere? Does not seem to be part of https://github.com/ethereum/execution-apis/pull/517, should reference all spec (or other) sources in description of PR

Reply:
> Is this format defined somewhere? Does not seem to be part of [ethereum/execution-apis#517](https://github.com/ethereum/execution-apis/pull/517), should reference all spec (or other) sources in description of PR

There is no spec for this. I think people had this discussion in the PR and [voted](https://github.com/ethereum/execution-apis/pull/517#issuecomment-1918512560) to have 2-bytes commit.